src/BitmovinEncodingLambda/index.js

```diff
@@ -1,3 +1,32 @@
+import mysql from 'mysql';
+
+function storeBitmovin(input, folder) {
+  const livestream_dir = 'bitmovin/' + folder
+  console.log('livestream_dir', livestream_dir)
+  console.log(`input: ${input}`);
+  updateAsset(input, livestream_dir);
+}
+
+function updateAsset(input, livestream_dir) {
+  const connection = mysql.createConnection({
+    host: process.env.DB_HOST,
```

Comment:
this is now using .env file, don't you need to make some changes in package.json file? also, update the doc on how to deploy this?